app/src/main/java/woowacourse/shopping/Modules.kt

```diff
@@ -0,0 +1,49 @@
+package woowacourse.shopping
+
+import org.koin.core.module.dsl.viewModel
+import org.koin.core.qualifier.named
+import org.koin.dsl.module
+import woowacourse.shopping.data.CartProductDao
+import woowacourse.shopping.data.CartRepository
+import woowacourse.shopping.data.DataBaseCartRepository
+import woowacourse.shopping.data.InMemoryCartRepository
+import woowacourse.shopping.data.ProductRepository
+import woowacourse.shopping.data.ProductRepositoryImpl
+import woowacourse.shopping.data.ShoppingDatabase
+import woowacourse.shopping.ui.MainViewModel
+import woowacourse.shopping.ui.cart.CartActivity
+import woowacourse.shopping.ui.cart.CartViewModel
+import woowacourse.shopping.ui.cart.DateFormatter
+
+val databaseModule =
+    module {
+        single<ShoppingDatabase> {
+            ShoppingDatabase.getInstance(get())
+        }
+        single<CartProductDao> { get<ShoppingDatabase>().cartProductDao() }
+    }
+
+val repositoryModule =
+    module {
+        single<CartRepository>(named("Database")) { DataBaseCartRepository(get()) }
```

Comment:
```suggestion
        single<CartRepository>(qualifier<DataBaseCartRepository>()) { DataBaseCartRepository(get()) }
```

qualifier<T>() 함수를 활용하면 하드코딩하지 않아도 됩니당!